### PR TITLE
Merge Improved Feature to master

### PR DESCRIPTION
**Pull request by ttxking**

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -4,35 +4,42 @@
...
     if request.method == 'POST':
         profile_form = ProfileForm(request.POST, request.FILES)
         if profile_form.is_valid():
             if user_profile.user == request.user and request.user.is_authenticated:
+                if profile_form.cleaned_data['profile_pic']:
+                    user_profile.profile_picture = profile_form.cleaned_data['profile_pic']
                 user_username = profile_form.cleaned_data['user_name']
                 user_profile.user.username = user_username
                 user_profile.user.email = profile_form.cleaned_data['email']
                 user_profile.user.first_name = profile_form.cleaned_data['first_name']
```
Comment (ttxking) [inline users/views.py]: We don't update first name

**Review by nicenicegame — APPROVED** [review]
Tested. Every features work fine.

**Review by bemyXmas — APPROVED** [review]
Pulled and tested the latest version of the improving feature.
- Still can't test if Twitter and Facebook social-auth can use or not until deployed
But other feature is working fine LGTM!

**Review by Kuea666 — APPROVED** [review]
I pulled it to test and didn't find any problems. Gj

**Pull request merged**